src/tribler/core/components/torrent_checker/torrent_checker/torrent_checker.py

```diff
@@ -155,15 +155,15 @@ async def check_random_tracker(self):
             response = await self.get_tracker_response(session)
         except Exception as e:  # pylint: disable=broad-except
             self._logger.warning(e)
+            # TODO: should we update the number of seeders to zero here?
```

Comment:
I'm not sure. An exception here shows that the tracker is unreachable. Which says nothing about the torrent health itself.

Reply:
I agree, I removed the comment